tests/Games/NexusMods.Games.FOMOD.Tests/NexusMods.Games.FOMOD.Tests.csproj

```diff
@@ -18,5 +18,21 @@
         <None Update="TestCasesPacked\**">
             <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
         </None>
+
+        <None Update="TestCases\NestedWithImages\random subfolder name\fomod\g1p1i1.png">
+          <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+        </None>
+
+        <None Update="TestCases\NestedWithImages\random subfolder name\fomod\g1p2i1.png">
+          <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+        </None>
+
+        <None Update="TestCases\NestedWithImages\random subfolder name\fomod\ModuleConfig.xml">
+          <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+        </None>
+
+        <None Update="TestCases\NestedWithImages\random subfolder name\fomod\moduleTitle.png">
+          <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+        </None>
```

Comment:
Similar to `TestCasesPacked`, you can use a glob for this:

```xml
<None Update="TestCases\**">
    <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
</None>
```

Reply:
I think these might be leftovers that were done automatically from when I created the new test archives. I'll remove it.